src/Common/Auth/OpenIDConnect/Entities/ClientEntity.php

```diff
@@ -68,13 +68,49 @@ class ClientEntity implements ClientEntityInterface
     const DSI_TYPE_EVIDENCE = 1;
     const DSI_TYPE_PREDICTIVE = 2;
 
+    protected string $identityProvider;
+    protected ?string $googleClientId;
+    protected ?string $googleClientSecret;
 
     public function __construct()
     {
         $this->scopes = [];
         $this->skipEHRLaunchAuthorizationFlow = false;
         $this->isEnabled = false;
         $this->dsiType = self::DSI_TYPE_NONE;
+        $this->identityProvider = 'local';
+        $this->googleClientId = null;
+        $this->googleClientSecret = null;
+    }
```

Comment:
IMO properties that can be set at the class should not be set in the constructor. Not only does PHP not have the same issue with mutables as Python does, but all properties are instance properties unless you explicitly use the `static` keyword. So I don't think this class needs a constructor at all.